pandas/tests/series/test_analytics.py

```diff
@@ -554,6 +554,10 @@ def test_count(self, datetime_series):
         ts.iloc[[0, 3, 5]] = np.nan
         tm.assert_series_equal(ts.count(level=1), right - 1)
 
+        # GH29478
+        with pd.option_context("use_inf_as_na", True):
+            pd.Series([pd.Timestamp('1990/1/1')]).count()
```

Comment:
Can you test the result of this expression?

Reply:
Updated, thanks a lot!